src/index.ts

```diff
@@ -3,15 +3,22 @@ import { loadStripe, Stripe } from '@stripe/stripe-js'
 import Axios from 'axios'
 import { recoverTypedSignature_v4 } from 'eth-sig-util'
 import * as ethers from 'ethers'
+import { _TypedDataEncoder } from 'ethers/lib/utils'
 import {
   Bid,
   DefaultApiFactory as DefaultApiFactoryV2,
   SignatureType,
   TokenERC721,
   WalletAddressProfile,
-  InlineObject1UserResidenceEnum,
+  InlineObject3UserResidenceEnum,
```

Comment:
Let's define this as a component in `api.yaml`.
This name will be changed.

Reply:
okay